examples/example4.json.txt

```diff
@@ -76,8 +76,8 @@ h'00112233445566778899aabbccddeeff0123456789abcdeffedcba9876543210'
 h'0123456789abcdeffedcba987654321000112233445566778899aabbccddeeff'
                 ]),
                 / image-size / 14:76834,
+                / compression-info / 19:bstr .cbor ({1:1 / "gzip"}) /,
```

Comment:
Can't parse that

Reply:
I'm pretty sure it's not quite correct. Maybe @bremoran can help?

Reply:
Maybe there's a better way to express what this means. What you'd normally see here is:

```
/ compression-info / 19 : h'A10101' / {
  \ suit-compression-algorithm \ 1 : 1 \ gzip \
} /
```

This is quite similar to what you see in [RFC8152](https://datatracker.ietf.org/doc/html/rfc8152#appendix-C.1.1):

```
/ protected / h'a10126' / {
  \ alg \ 1:-7 \ ECDSA 256 \
} / ,
```
However this appears so much in SUIT and the bstrs are so large that this quickly becomes unmanagable. I proposed this notation at IETF108, and there was support for simplifying the printing, however it looks like I had an action item to adopt CDDL's version of diagnostic notation for encoded CBOR items that I didn't follow up. I'll make a PR that adopts https://datatracker.ietf.org/doc/html/rfc8610#appendix-G.3 so that this is more readable.

Reply:
See #34